helpers/misk.sh

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2019 Denis Gavrilyuk <karpa4o4@gmail.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# Compress an file.
+# Globals:
+#     PIEMAN_UTILS_DIR
+#     PYTHON
+# Arguments:
+#     Compress type
+#     File
+# Returns:
+#     None
+compress_file() {
+  local compress_type=${1}
+  local file=${2}
+
+  ${PYTHON} "${PIEMAN_UTILS_DIR}"/compress_file.py "${compress_type}" "${file}"
+}
+
+# Decompress an file.
+# Globals:
+#     PIEMAN_UTILS_DIR
+#     PYTHON
+# Arguments:
+#     Decompress type
+#     File
+# Returns:
+#     None
+decompress_file() {
+  local decompress_type=${1}
+  local file=${2}
+
+  ${PYTHON} "${PIEMAN_UTILS_DIR}"/compress_file.py -d "${decompress_type}" "${file}"
+}
+
+# Wget analog writed on Python.
```

Comment:
No, actually this is a wrapper around `wget.py` which, in turn, is an analog of wget, written in Python.
I guess the most suitable description is:
> Downloads files from the Web non-interactively.